### PR TITLE
sync Travis CI with GitHub

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

**Comment by jhoblitt** [comment]
A PR title would be helpful.  Did you want this to run periodically or be manually triggered?

File: jobs/run_travissync.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import util.Common
+Common.makeFolders(this)
+
+pipelineJob('travissync/travissync') {
```
Comment (jhoblitt) [inline jobs/run_travissync.groovy]: Do we need a dedicated top level folder for this job?  People are complaining about too much clutter at the top level so I'd prefer to at least us a generic name.

**Comment by athornton** [comment]
Run once an hour, please.

**Comment by jhoblitt** [comment]
Take a look at https://github.com/lsst-sqre/jenkins-dm-jobs/blob/master/jobs/weekly_release.groovy#L16-L18as an example of running a build periodically.

**Comment by athornton** [comment]
I will go add a trigger.

File: jobs/run_travissync.groovy [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  label('travissync')
+  keepDependencies(true)
+  concurrentBuild()
+
+  def repo = SEED_JOB.scm.userRemoteConfigs.get(0).getUrl()
+  def ref  = SEED_JOB.scm.getBranches().get(0).getName()
+
+  triggers {
+    cron('17 * * * *')
```
Comment (jhoblitt) [inline jobs/run_travissync.groovy]: Using `H` in place of `17` is slightly preferable as it allows jenkins to choose a random skew.

**Pull request merged**